packages/components/src/DateTimePickers/DateRange/Inputs/Inputs.component.js

```diff
@@ -0,0 +1,82 @@
+import React, { useContext, useEffect, useRef } from 'react';
+import omit from 'lodash/omit';
+import DebounceInput from 'react-debounce-input';
+
+import { DateRangeContext } from '../Context';
+import { START_DATE, END_DATE } from '../constants';
+import InputSizer from '../../shared/InputSizer';
+
+import Icon from '../../../Icon';
+
+import theme from './Inputs.scss';
+
+const OMIT_PROPS_INPUT = ['startInputRef', 'endInputRef'];
+
+function Inputs(props) {
+	const { startDate, endDate, inputManagement } = useContext(DateRangeContext);
+	const { placeholder, onChange, onFocus, focusedInput } = inputManagement;
+
+	let startDateInputRef = useRef(null);
+	let endDateInputRef = useRef(null);
+
+	useEffect(() => {
+		if (startDateInputRef && focusedInput === START_DATE) {
+			startDateInputRef.focus();
+		}
+		if (endDateInputRef && focusedInput === END_DATE) {
+			endDateInputRef.focus();
+		}
+	});
+
+	return [
+		<InputSizer placeholder={placeholder} inputText={startDate.value ? startDate.textInput : ''}>
+			{width => (
+				<DebounceInput
```

Comment:
The labels are missing

Reply:
by labels do you mean placeholder or "from"/"to"?

Reply:
Yes the labels should be set with the inputs, and don't forget i18n

Reply:
labels added.
I thought the labels will be provided by form schema.
So talking about form wrapper of this, we just need to wrap `InputDateRangePicker`? instead of composing 2 DateWidget?

Reply:
There would be 2 kinds of labels. For example, when you define travel dates you will have some kind of "Travel", then "from/to".
We will have to manage the labels a little differently to have screen readers have some context when focusing on the inputs, let's tackle that with the wrapper.